advanced-vector-search/README.md

```diff
@@ -247,21 +257,25 @@ If you want to run the image with `annoy` as a search library, you can override
 docker run -it --entrypoint=/bin/bash jinahub/app.example.advancedvectorsearch:0.0.2-0.9.20 entrypoint.sh annoy
 ```
 
-If you want to change the parameters or of the `Faiss` or the `Annoy` Indexer you can pass different environment variables
+If you want to change the parameters of `Faiss` or `Annoy` Indexer you can pass different environment variables
 to the `docker run` command by doing for instance:
 
 ```bash
 docker run -e JINA_FAISS_INDEX_KEY='Flat' jinahub/app.example.advancedvectorsearch:0.0.2-0.9.20
 ```
 
-An important parameter to set is `JINA_DISTANCE_REVERSE`, depending on the type of distance or metric that is used. For instance
-for `inner_product` distance, `JINA_DISTANCE_REVERSE` should be set to True as the returned measure is similarity for `Faiss` and not
-`distance`. Therefore the results should be scores in descending order.
+## Good to know
+
+An important parameter to set is `JINA_DISTANCE_REVERSE`, depending on the type of distance or metric that is used. 
+For instance for `inner_product` distance, `JINA_DISTANCE_REVERSE` should be set to `True`.
+This is because returned measure for `Faiss` is similarity and not distance. 
+Which means that the results should be scored in descending order to get what we would expect.
```

Comment:
should be sorted in descending order

Reply:
Ah yes, I wrote "scored" instead of "sorted". Got it